db/sql/sql.go

```diff
@@ -505,14 +505,24 @@ func makeColumn(tableName string, property schema.Property) string {
 }
 
 // MakeColumns generates an array that has Gohan style colmun names
-func MakeColumns(s *schema.Schema, tableName string, join bool) []string {
+func MakeColumns(s *schema.Schema, tableName string, join bool, aliasTableName string) []string {
 	var cols []string
 	manager := schema.GetManager()
 	for _, property := range s.Properties {
-		cols = append(cols, makeColumn(tableName, property)+" as "+quote(makeColumnID(tableName, property)))
+        if aliasTableName != "" {
```

Comment:
I believe we can do this always.

 newAliasTableName = aliasTableName + "__" + property.RelationProperty